### ISSUE TITLE
Jump to main content link doesn't work

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/notification-planning#2109]

**Comment by yaelberger-commits** [comment]
We could do some more testing on Windows to try and reproduce the bug.

**Comment by yaelberger-commits** [comment]
Andrew says we could be missing an anchor for main content

**Comment by andrewleith** [comment]
Waiting on PR review.
